spec/acceptance/machine_spec.rb

```diff
@@ -0,0 +1,64 @@
+require 'spec_helper_acceptance'
+
+describe 'docker::machine' do
+  context 'with default parameters' do
+    pp = <<-EOS
+      include docker::machine
+    EOS
+
+    it 'applies with no errors' do
+      apply_manifest(pp, catch_failures: true)
+    end
+
+    it 'is idempotent' do
+      apply_manifest(pp, catch_changes: true)
+    end
+
+    describe file('/usr/local/bin/docker-machine-0.16.1') do
+      its('owner') { is_expected.to eq 'root' }
+      its('mode') { is_expected.to cmp '0755' }
+      it { is_expected.to be_file }
```

Comment:
Could you please try updating the tests with the similar changes given below https://rspec-puppet.com/tutorial/   
 is_expected.to contain_file('/etc/logrotate.d/nginx').with({
      'ensure' => 'present',
      'owner'  => 'root',
      'group'  => 'root',
      'mode'   => '0444',
    })

Please let me know if it doesn't work

Reply:
Hey @sheenaajay, sorry but I won't play Github comment ping pong for failing tests. If I'm not able to run the tests locally/on Travis without your intervention I won't build them because it just takes way to much time.

Reply:
@baurmatt Yeah, Let me make the changes and run on our Jenkins.

Reply:
Thanks! :)